src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossSummary.scala

```diff
@@ -87,10 +105,23 @@ object TreeLossSummary {
         val areaHa = area / 10000.0
 
         val gainArea: Double = gain * areaHa
+
         val biomassPixel = biomass * areaHa
-        val agc2000Pixel = agc2000 * areaHa
-        val bgc2000Pixel = bgc2000 * areaHa
-        val soilCarbon2000Pixel = soilCarbon2000 * areaHa
+
+        val agc2000Pixel: Double = if (getAnyMapValue[Boolean](kwargs, "carbonPools"))
+          agc2000 * areaHa
```

Comment:
Use the 'wantCarbonPools' val from above here and following as well.

Reply:
Done.